tests/testlib.h

```diff
@@ -46,4 +46,11 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC (TestsStdoutToStderr, tests_stdout_to_stderr_end);
 #define TESTS_SCOPED_STDOUT_TO_STDERR \
   G_GNUC_UNUSED g_autoptr(TestsStdoutToStderr) _tests_stdout_to_stderr = tests_stdout_to_stderr_begin ()
 
+/* Implemented as a macro so we get useful line numbers in assertion failures */
+#define assert_cmpstr_free_lhs(owned_string, op, unowned_string) \
```

Comment:
`op` unused?

Reply:
Oops, yes: it was blindly assuming `==`. Now fixed.